etna/lib/etna/application.rb

```diff
@@ -91,6 +91,10 @@ def environment
     (ENV["#{self.class.name.upcase}_ENV"] || :development).to_sym
   end
 
+  def id
+    self.class.name.snake_case.split(/::/).last
```

Comment:
Hmm.  There is also the `APP_NAME` environment variable that I set in all application environments.  Would it make sense to prefer that value if set?

Reply:
> md5s can also be stub values and nil in metis, as they are calculated asynchronously by a background job. Does this PR test against those cases?

I didn't test against those cases, but the method is very lazy and will give up and say "null" if metis doesn't know the file exists (common in dev). If metis hasn't hashed the file yet you get back a `temp-` hash, which seems like the right thing to do.

Reply:
Err, to be clear I didn't test in specs, I tested it against my dev instance.